src/routes/trips.js

```diff
@@ -324,20 +324,181 @@ import validateOneWayTrip from '../middlewares/validateOneWay';
+               "in": "path",
+               "required": true,
+               "type": "string",
+               "description": "target key"
+             },
+           ],
+           "responses": {
+             "200": {
+               "description": "",
+               "headers": {}
+             }
+           }
+         }
+       }
+    */
+/**
+ * @swagger
+ *  "/trips/edit/": {
+      "patch": {
+        "description": "A user can edit an open request.",
+        "summary": "Edit a request.",
+        "tags": [
+          "Trips"
+        ],
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/x-www-form-urlencoded"
+        ],
+        "parameters": [
+          {
+              "name": "requestId",
+              "in": "query",
+              "description": "ID of a request to be updated",
+              "required": true,
+              "type": "integer",
+              "format": "int64"
+          },
+          {
+            "name": "origin",
+            "in": "formData",
+            "type": "string",
+          },
+          {
+            "name": "destination",
+            "in": "formData",
+            "type": "string",
+          },
+          {
+            "name": "departureDate",
+            "in": "formData",
+            "type": "string",
+            "value": "YYYY-MM-DD"
+          },
+          {
+            "name": "returnDate",
+            "in": "formData",
+            "type": "string",
+            "value": "YYYY-MM-DD"
+          },
+          {
+            "name": "accommodation",
+            "in": "formData",
+            "type": "string",
+          },
+          {
+            "name": "reason",
+            "in": "formData",
+            "type": "string",
+          },
+          {
+            "name": "passportNumber",
+            "in": "formData",
+            "type": "string",
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success!",
+          },
+          "422": {
+            "description": "Invalid input",
+          },
+          "404": {
+              "description": "Not found",
+            }
+        }
+      }
+    }
+ */
 import requestsController from '../controllers/searchRequest';
 import validateInputs from '../middlewares/validateReturnTrip';
 import comment from '../controllers/comment.controller';
 import commentValidate from '../middlewares/newComment';
 import checkCommenterValidation from '../middlewares/commenter';
 import requestStatusController from '../controllers/notifications';
+import splitUrl from '../middlewares/splitUrl';
 
 import {
   validateRequestDate, validateCityDate, tripInformation, multicity, checkIfRequestExists
 } from '../middlewares/validateDate';
 
 
 const router = express.Router();
+
+const {
+  createTwoWayTrip, pendingApproval, rejectRequest, createMultiCityRequest, editRequest
+} = requestControllers;
```

Comment:
I don't find a reason to destructure on this line while you could destructure directly in the import.

Reply:
Correction made. 
Thanks @kagabof